src/theory/bv/theory_bv_utils.h

```diff
@@ -125,13 +125,30 @@ Node mkSortedNode(Kind kind, std::vector<Node>& children);
 
 /* Create node of kind NOT. */
 Node mkNot(Node child);
+
 /* Create node of kind AND. */
 Node mkAnd(TNode node1, TNode node2);
-Node mkAnd(const std::vector<TNode>& conjunctions);
-Node mkAnd(const std::vector<Node>& conjunctions);
+/* Create n-ary node of kind AND. */
+template<bool ref_count>
+Node mkAnd(const std::vector<NodeTemplate<ref_count>>& conjunctions)
+{
+  std::set<TNode> all;
+  all.insert(conjunctions.begin(), conjunctions.end());
```

Comment:
Nit: the two statements above can actually be combined into `std::set<TNode> all(conjunctions.begin(), conjunctions.end());`

Reply:
Done